src/odb/include/odb/db.h

```diff
@@ -7456,9 +7480,58 @@ class dbMetalWidthViaMap : public dbObject
   // User Code End dbMetalWidthViaMap
 };
 
+class dbModBTerm : public dbObject
+{
+ public:
+  const char* getName() const;
+
+  dbModule* getParent() const;
+
+  void setNet(dbModNet* net);
+
+  dbModNet* getNet() const;
+
+  void setNextNetModbterm(dbModBTerm* next_net_modbterm);
+
+  dbModBTerm* getNextNetModbterm() const;
+
+  void setPrevNetModbterm(dbModBTerm* prev_net_modbterm);
+
+  dbModBTerm* getPrevNetModbterm() const;
+
+  void setNextEntry(dbModBTerm* next_entry);
+
+  dbModBTerm* getNextEntry() const;
+
```

Comment:
Should be private and not in db.h

Reply:
automatically generated code. Is there some other way of controlling the code generation. It is automatically generated and put in db.h

Reply:
See "flags" : ["private"] in other classes